helm/prometheus-rules/templates/alerting-rules/flux.rules.yml

```diff
@@ -0,0 +1,63 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  creationTimestamp: null
+  labels:
+    {{- include "labels.common" . | nindent 4 }}
+  name: fluxcd.rules
+  namespace: {{ .Values.namespace  }}
+spec:
+  groups:
+  - name: fluxcd
+    rules:
+    - alert: FluxHelmReleaseFailed
+      annotations:
+        description: |-
+          {{`Flux HelmRelease on {{ $labels.installation }}/{{ $labels.cluster_id }} in {{ $labels.namespace }}/{{ $labels.name }} is stuck in Failed state.`}}
+        opsrecipe: fluxcd/failing
```

Comment:
I added an ops recipe for helmrelease problems. Can we link to that somehow?

Reply:
Adding the link would help 🤦‍♂️ here it is

https://intranet.giantswarm.io/docs/support-and-ops/ops-recipes/fluxcd-helmrelease-not-ready/

Reply:
I've just seen it, it's so helpful! I'll redirect alert for `HelmReleases` to your recipe.

Reply:
Thx!